lane.py

```diff
@@ -1157,7 +1158,6 @@ def search(self, query, search_client, pagination=None):
                         lazyload(mw.license_pool, LicensePool.identifier),
                         lazyload(mw.license_pool, LicensePool.presentation_edition),
                     )
-                    q = self.only_show_ready_deliverable_works(q, mw)
```

Comment:
This reminds me that I need to update the search index when someone suppresses a book through the admin interface. Once I do that, the search index shouldn't return suppressed books anyway. We could still apply the filter just in case. The only part of it that I left out of the search index filter was DeliveryMechanism.default_client_can_fulfill. Clients should be able to figure out that there's another page of search results if they see a page with only books they can't fulfill.